### PR TITLE
Enable unit testing

### LINKED ISSUE #45: Unit testing
We should add a platform for executing unit tests. Writing tests for all new and changed functionality should also become our contribution policy.

### PR DESCRIPTION
**Pull request by ForNeVeR**
I've created sample test and enabled testing using utest library. Closes #45.

### CODE REVIEW

**Comment by ForNeVeR** [comment]
I choose mr. @rexim as a unit test expert for today.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: scala
 jdk: oraclejdk8
 scala:
-  - 2.11.5
-script: sbt site
+  - 2.11.6
+script: sbt test && sbt site
```
Comment (rexim) [inline .travis.yml]: Can't you just `sbt test site`?

**Comment by rexim** [comment]
I like the shit. Approved!

File: .travis.yml (hunk 1 above)
Comment (ForNeVeR) [inline .travis.yml]: Yeah, I'll replace it. Thanks.

**Pull request merged**